openassessment/xblock/static/sass/oa/views/_oa-base.scss

```diff
@@ -99,6 +99,11 @@
       width: 100%;
       margin-bottom: 0;
 
+      button.ui-slidable {
+        @include text-align(left);
+        display:flex;
```

Comment:
@diegomillan Formatting: Missing space after colon. The same note applies to L190, L197, L205, and L213 below.